.github/workflows/release.yaml

```diff
@@ -26,6 +26,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
```

Comment:
```suggestion
      - uses: actions/checkout@master
        with:
          fetch-depth: 0

      - run: git fetch origin +refs/tags/*:refs/tags/*
```

Reply:
No need, it's pulling the tags on the checkout step

Reply:
Ok, I had issues with that on subctl, may be they changed the checkout action?